src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -729,26 +784,29 @@ static Library()
             {
                 BuiltinFunctionsCore.Ln,
                 StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Ln.Name,
                     expandArguments: NoArgExpansion,
                     replaceBlankValues: ReplaceBlankWithZero,
                     checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
-                    checkRuntimeValues: StrictNumericPositiveNumberChecker,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
```

Comment:
We should still do Positive number checks here right?

Reply:
There is no need for an additional check - the function will return a non-finite number, and the wrapper will create the appropriate error